packages/cactus-plugin-ledger-connector-fabric/src/test/typescript/integration/fabric-v2-2-x/deploy-cc-from-golang-source-private-data.test.ts

```diff
@@ -39,361 +37,356 @@ import { DiscoveryOptions } from "fabric-network";
-  };
-
-  //Chaincode-specific method requires attribute asset_properties
-  const rawTmpData = {
-    asset_properties: assetData,
-  };
-
-  // CreateAsset(id string, color string, size int, owner string, appraisedValue int)
-  const createRes = await apiClient.runTransactionV1({
-    transientData: rawTmpData,
-    contractName,
-    channelName,
-    //objectType, assetID, color, size, appraisedvalue
-    params: [],
-    methodName: "CreateAsset",
-    invocationType: FabricContractInvocationType.Sendprivate,
-    signingCredential: {
-      keychainId,
-      keychainRef: keychainEntryKey,
-    },
+  afterAll(async () => {
+    const pruning = pruneDockerAllIfGithubAction({ logLevel });
+    await expect(pruning).resolves.toBeTruthy();
   });
-  t.ok(createRes, "setRes truthy OK");
-  t.true(createRes.status > 199, "createRes status > 199 OK");
-  t.true(createRes.status < 300, "createRes status < 300 OK");
-  t.comment(`BassicAssetTransfer.Create(): ${JSON.stringify(createRes.data)}`);
-
-  const getRes = await apiClient.runTransactionV1({
-    contractName,
-    channelName,
-    params: [assetId],
-    methodName: "ReadAsset",
-    invocationType: FabricContractInvocationType.Send,
-    signingCredential: {
-      keychainId,
-      keychainRef: keychainEntryKey,
-    },
+  beforeAll(async () => {
+    await ledger.start();
+
+    const listenOptions: IListenOptions = {
+      hostname: "localhost",
+      port: 0,
+      server,
+    };
+    addressInfo = (await Servers.listen(listenOptions)) as AddressInfo;
+    ({ port } = addressInfo);
+    apiUrl = `http://localhost:${port}`;
+
+    configuration = new Configuration({ basePath: apiUrl });
+    apiClient = new FabricApi(configuration);
   });
 
-  t.ok(getRes, "getRes truthy OK");
-  t.ok(getRes.data, "getRes.data truthy OK");
-  t.ok(getRes.data.functionOutput, "getRes.data.functionOutput truthy OK");
-  t.true(getRes.status > 199 && createRes.status < 300, "getRes status 2xx OK");
-  t.comment(`ReadAsset ResponseBody: ${JSON.stringify(getRes.data)}`);
-
-  //TODO FIX:
-  //Error: failed to read asset details: GET_STATE failed: transaction ID: 0a41ae425e259ee6c1331d4d3c06bd9fc4727f9961abc0c1a2895c450fc8411a: tx creator does not have read access permission on privatedata in chaincodeName:asset-transfer-private-data collectionName: Org2MSPPrivateCollection
-  //This has probably to do with the state database supported by Fabric test ledger
-  /*
-  const collectionToParse = "Org1MSPPrivateCollection";
-  const getResPrivate = await apiClient.runTransactionV1({
-    contractName,
-    channelName,
-    params: [collectionToParse, assetId],
-    methodName: "ReadAssetPrivateDetails",
-    invocationType: FabricContractInvocationType.SEND,
-    signingCredential: {
-      keychainId,
-      keychainRef: keychainEntryKey,
-    },
-  });
+  test(testCase, async () => {
+    const channelId = "mychannel";
+    const channelName = channelId;
+
+    // test.onFailure(async () => {
```

Comment:
@awadhana Please delete commented out code that is no longer needed/used in general.